configs/components/rubygem-aws-partitions.rb

```diff
@@ -1,6 +1,15 @@
 component "rubygem-aws-partitions" do |pkg, settings, platform|
-  pkg.version "1.960.0"
-  pkg.md5sum "c8b0c649c7a00c79e0af4272debbd8f9"
+  version = settings[:rubygem_aws_partitions_version] || '1.961.0'
+  pkg.version version
+
+  case version
+  when '1.961.0'
```

Comment:
Not sure why we would need separate versions for this. This gem is updated very frequenty, perhaps there was an update to it between your gem installs. I believe these versions are compatable with both ruby 2/3

Reply:
Removed the case but just updated to the most recent version